docs/reference/ssl/swarm-with-traefik.md

```diff
@@ -0,0 +1,180 @@
+# SSL on Swarm with Traefik
+
+To completely secure your OpenFaaS installation, you need SSL.  On Swarm, you can do this easily with [Traefik][traefik] and [Let's Encrypt][letsencrypt].  Traefik is a reverse proxy that comes with SSL support via Let's Encrypt. In this tutorial we will show you how to deploy OpenFaaS with Traefik.
```

Comment:
"To completely secure your OpenFaaS installation, you need SSL" - I'm worried that this is a bit too optimistic. 

How about something more specific / description, i.e. about enabling encryption between the gateway and the outside world? Why is that important?

Reply:
An explanation of what and why you should use TLS is not included in the k8s document